xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/saverights.vm

```diff
@@ -27,11 +27,28 @@
 #set($foundd = false)
 
 #if ($!{services.csrf.isTokenValid("$!{request.getParameter('form_token')}")})
+## Convert the user or group identifier present in the request to a DocumentReference relative to the current wiki
+## if no wiki is specified.
+#set ($userOrGroupReference = $services.model.resolveDocument($fullname, 'current'))
```

Comment:
```suggestion
#set ($userOrGroupReference = $services.model.resolveDocument($fullname))
```

``current`` is already the default resolver used [it seems](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/script/ModelScriptService.java#L69).